tools/debug/debugsymbolviewer.py

```diff
@@ -169,7 +169,7 @@ def parse_file(log_file):
 
     setup_symbol_table(systemmap_path)
 
-    print('----------------------------------  DEBUG DISPLAY SYMBOLS -------------------------------')
+    print('------------------------------- DEBUG SYMBOLS IN TEXT RANGE -----------------------------')
```

Comment:
I think previous `DEBUG DISPLAY SYMBOLS` is enough.
Is there any special reason to print `TEXT RANGE`?

Reply:
Kishore had mentioned in one of the comments that its not clear which symbols are these exactly, hence renamed them